lldap_config.docker_template.toml

```diff
@@ -7,9 +7,17 @@
 ## You can set it with the LLDAP_VERBOSE environment variable.
 # verbose=false
 
+## The host address that the LDAP server will be bound to.
+## To enable IPv6 support, simply switch "ldap_host" to "::1":
```

Comment:
Can you add something like:

"To only allow connections from localhost (if you want to restrict to local self-hosted services), change it to "127.0.0.1" ("::1" in IPv6)"

And yes, btw, "::1" is localhost, you want to suggest "::" for the general case for IPv6.

Reply:
> Can you add something like:
> 
> "To only allow connections from localhost (if you want to restrict to local self-hosted services), change it to "127.0.0.1" ("::1" in IPv6)"

Done: https://github.com/nitnelave/lldap/pull/377/commits/578d5e1fdb5c407f33c2d46f04519f20dd74c22d

> And yes, btw, "::1" is localhost, you want to suggest "::" for the general case for IPv6.

yep, sure thx. did correct it.